django/db/models/expressions.py

```diff
@@ -995,14 +995,15 @@ def set_source_expressions(self, exprs):
     def get_source_expressions(self):
         return [self.expression]
 
-    def as_sql(self, compiler, connection):
+    def as_sql(self, compiler, connection, **kwargs):
```

Comment:
This method should actually mix `placeholders` with `**kwargs`, which expands the usefulness of `**kwargs` to the `ordering` interpolation variable, but also any other interpolation variables that might be introduced in a subclass.